### PR TITLE
Closes #3: Trigger Safe transaction to enable protocol and module

### LINKED ISSUE #3: Trigger Safe transaction to add a new plugin
- Enable specified plugin on the Safe Protocol Manager
- If Safe Protocol Manager is not enable also enable it as a module on the Safe

### PR DESCRIPTION
**Pull request by rmeissner**

### CODE REVIEW

**Review by akshay-ap — APPROVED** [review]

File: contracts/src/utils/protocol.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ export const getProtocolManagerAddress = async(hre: HardhatRuntimeEnvironment):
 
     // For the tests we deploy a mock for the manager
     if (chainId === "31337") return deployMock(hre, "ManagerMock")
+
+    if (chainId === "5") return "0xb4Dc1B282706aB473cdD1b9899c57baD2BD3e2f3"
```
Comment (akshay-ap) [inline contracts/src/utils/protocol.ts:25]: Actually, this is the test manager contract: `0xAbd9769A78Ee63632A4fb603D85F63b8D3596DF9`

https://goerli.etherscan.io/address/0xAbd9769A78Ee63632A4fb603D85F63b8D3596DF9#code

https://github.com/5afe/safe-core-protocol/blob/main/deployments.ts#L1311

File: web/src/logic/protocol.ts [hunk 2]
```diff
@@ -11,11 +11,24 @@ const PLUGIN_ABI = [
     "function metadataProvider() external view returns (uint256 providerType, bytes location)"
 ]
 
+export const getManager = async() => {
+    const provider = await getProvider()
+    const registryInfo = {
+        address: "0xb4Dc1B282706aB473cdD1b9899c57baD2BD3e2f3",
```
Comment (akshay-ap) [inline web/src/logic/protocol.ts:17]: Test registry address: `0x9EFbBcAD12034BC310581B9837D545A951761F5A`
https://github.com/5afe/safe-core-protocol/blob/a1b0da08b251b1f7aa6d1ae09b4fc6b474813979/deployments.ts#L1054

**Review by akshay-ap — APPROVED** [review]
Changes needed but approved so that it can be merged with fixes without blockers

File: web/src/logic/protocol.ts (hunk 2 above)
Comment (rmeissner) [reply web/src/logic/protocol.ts:17]: I will update this in a follow up PR

**Pull request merged**